eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventData.java

```diff
@@ -4,13 +4,8 @@
 package com.azure.eventhubs;
 
 import com.azure.core.amqp.MessageConstant;
```

Comment:
With JMS potentially being added to the Messaging offerings, do you want to expose the underlying AMQP here?

Reply:
Good observation. I created an issue for future Connie to think about. https://github.com/Azure/azure-sdk-for-java/issues/3885